setup.py

```diff
@@ -439,7 +452,10 @@ def run(self) -> None:
         for dir_to_delete in self.dir_list_to_delete:
             dir_delete = pathlib.Path(dir_to_delete)
             if dir_delete.exists():
-                dir_util.remove_tree(str(dir_delete))
+                try:
+                    shutil.rmtree(dir_delete)
+                except:
+                    print('Error deleting directory')
```

Comment:
Log directory to delete

Reply:
do we have a logger where we are flashing all these logs or the standard out gets piped to somewhere else by default